data/p02369.c

```
@@ -85,18 +85,18 @@ void push_edge(struct adjcency_list* graph,
 	       struct edge _edge) {
 
     struct edge_list_node *n,*p;
-    
+
     n = (struct edge_list_node*)malloc(sizeof(struct edge_list_node));
     n->e = _edge;
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)